### ISSUE TITLE
Add support to multiple pdf files

### ISSUE THREAD

**Issue by AlessandroZavoli** [SonicWarrior1/pdfchat#7]
Hi, it is possible to extend this approach to handle a small collection (1-50) of pdf files?

**Comment by SonicWarrior1** [comment]
Yes it can surely be extended to support multiple pdf files but then we have to extend the context length for the llm and might also have to change the retrieval technique.

**Comment by 25mb-git** [comment]
I made changes to support multiple pdf files and to implement RAG architecture. https://github.com/25mb-git/pdfchat
